documentation.md

```diff
@@ -0,0 +1,25 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+_**If in doubt consult with your security point of contact**_
+
+## Restricted Items
```

Comment:
This list sets out examples of restricted items. It is not, and cannot be, exhaustive. We should state this clearly, as well as explicitly defining in words what types of information should be restricted and importantly why.